### ISSUE TITLE
Where is the test code? How to test after training?

### ISSUE THREAD

**Issue by longmalongma** [kentaroy47/vision-transformers-cifar10#20]
Where is the test code? How to test after training?@

**Comment by kentaroy47** [comment]
Since there is no dedicated test set in cifar10, the validation is often treated as the test set and is run every epoch

**Comment by longmalongma** [comment]
> Since there is no dedicated test set in cifar10, the validation is often treated as the test set and is run every epoch

@kentaroy47 But I use your code to get lower accuracy of vision transformer than resnet 18 on validation set? Why is this?

**Comment by kentaroy47** [comment]
if you train from scratch, vit does not get good accuracy.

if you use finetuning, it will surpass resnet18 accuracy.
